docs/guide/writing-tasks.md

```diff
@@ -513,27 +513,47 @@ publish_task:
 
     Names can be also pretty complex:
 
-
     ```yaml
     task:
-      name: test (linux)
-      ...
-
-    task:
-      name: test (windows)
-      ...
-
-    task:
-      name: test (macOS)
-      ...
-
+      name: Tests Shard $TESTS_SPLIT
+      env:
+        matrix:
+          TESTS_SPLIT: 1/3
+          TESTS_SPLIT: 2/2
+          TESTS_SPLIT: 3/3
+      tests_script: ./.ci/tests.sh
+    
     deploy_task:
+      only_if: $CIRRUS_BRANCH == 'master'
       depends_on:
-        - test (linux)
-        - test (windows)
-        - test (macOS)
+        - Tests Shard 1/3
+        - Tests Shard 2/3
+        - Tests Shard 3/3
+      script: ./.ci/deploy.sh
       ...
     ```
+    
+    Complex task names make it difficult to list and **maintain** all of such task names in your `depends_on` field. In order to 
+    make it simpler you can use the `alias` field to have a short simplified name for several tasks to use in `depends_on`.
+    
+    Here is a modified version of an example above that leverages the `alias` field:
+    
+    ```yaml hl_lines="3 13"
+    task:
+      name: Tests Shard $TESTS_SPLIT
```

Comment:
```suggestion
      name: Test Shard $TESTS_SPLIT
```
Test is an adjective

Reply:
Sorry, I was late to the party

Reply:
hang on a second
I missed something?!

😠 impossible

Reply:
Nope, seems names are consistent, for example: `Tests Shard 1/3`

Reply:
> Nope, seems names are consistent, for example: `Tests Shard 1/3`

It could be consistent, but it is not grammatically correct, That's what I mean. 
in "Test shard", shard is the noun, and test is the adjective, therefore it cannot be tests since adjetives cannot be expressed in plural.

At least that's how I interpret it. But if you mean "Tests - Shard 1/3" I would say that one is correct

Reply:
Yeah he's right, that's not grammatically correct